### PR TITLE
QoL and Learn standards alignment (Part 1 of n)

### PR DESCRIPTION
**Pull request by ckittel**
As part of my ramping on this project, and also part of making sure we're aligning to Microsoft Learn standards.  This is a pretty basic initial pass.  There will be more coming, but I'll do it in additional PRs.

There were no functional changes with this PR (intended) with the exception of:

* Removed the one PowerShell-based instruction (since everything else was just bash -- felt very out of place)

### CODE REVIEW

File: scenarios/aca-internal/bicep/README.md [hunk 1]
```diff
@@ -1,105 +1,124 @@
...
+   | `vnetAddressPrefixes` | An array of string. The address prefixes to use for the hub virtual network. | `["10.0.0.0/16"]` | `["10.100.0.0/16"]` |
+   | `enableBastion` | Controls if Azure Bastion is deployed. | `true` | false` |
+   | `bastionSubnetAddressPrefix` | CIDR to use for the Azure Bastion subnet. Must be a subset of the hub CIDR ranges. | **10.0.2.0/27** | **10.100.2.0/27** |
+   | `vmSize` | The size of the virtual machine to create for the jump box. | `Standard_B2ms` | Any one of: [VM sizes](https://learn.microsoft.om/azure/virtual-machines/sizes) |
+   | `vmAdminUsername` | The username to use for the jump box. | **azureuser** | `jumpboxadmin` |
+   | `vmAdminPassword` | The password to use for the jump box admin user. | **Password123**. :stop_sign: You *should* change this. | Any cryptographically strong password of your choosing. |
+   | `vmLinuxSshAuthorizedKeys` | The SSH public key to use for the jump box (if VM is linux) | *unusable/garbage value* | Any SSH keys you wish n the form of **ssh-rsa AAAAB6NzC...P38/oqQv description**|
```
Comment (ckittel) [inline scenarios/aca-internal/bicep/README.md:86]:
```suggestion
   | `vmLinuxSshAuthorizedKeys` | The SSH public key to use for the jump box (if VM is Linux) | *unusable/garbage value* | Any SSH keys you wish in the form of **ssh-rsa AAAAB6NzC...P38/oqQv description**|
```

**Review by thotheod — APPROVED** [review]
All changes look really good. Thank you very much for that.
I would like from your side to review a small change I 've done. The  SP or the user who deploys needs to have OWNER RBAC (not contributor) because the deployment creates managed Identities and assigns roles to resources (i.e. keyvault, ACR etc)

**Comment by ckittel** [comment]
Looks great!  Thanks for setting that straight.

**Comment by ckittel** [comment]
#sign-off

(I know that doesn't work in this repo)

**Pull request merged**